src/main/java/junit/App.java

```diff
@@ -1,4 +1,4 @@
-package org.example;
+package junit;
```

Comment:
Do we need this App.java class?
I suppose it should be deleted or cleaned up with comments and sout line.
If you want to leave this "Enter point" - leave it empty (main)